pom.xml

```diff
@@ -1578,6 +1578,7 @@
               <target>
                 <property name="mvn.project.build.directory" value="${project.build.directory}" />
                 <property name="mvn.runtime_classpath" refid="maven.runtime.classpath" />
+                <property name="java.version" value="${java.version}"/>
```

Comment:
The others are `mvn...`, so I would expect we follow the same pattern here.

Reply:
done